lib/rules.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'board'
+
+module Rules
```

Comment:
Out of curiosity, why is Rules a module and not a class? Not saying this is right or wrong, just curious.

Reply:
I chose to use it as a sort of static class since it is a small collection of rules methods.